hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -25,17 +72,10 @@ Prevent Popups to overflow tables.
       buttonOffset = $button.offset();
 
       $popup
-        .css('top', parseInt(buttonOffset.top, 10) + $button.outerHeight())
-        .css('left', parseInt(buttonOffset.left, 10) - ($popup.width() - $button.outerWidth()));
+        .css('top', +buttonOffset.top + $button.outerHeight())
+        .css('left', +buttonOffset.left - $popup.width() - $button.outerWidth());
```

Comment:
I can notice brackets in the previous version of the code and the operation here is subtraction:
```js
($popup.width() - $button.outerWidth())
```
But in the current change these brackets were removed. I think that actually breaks the formula

![image](https://user-images.githubusercontent.com/3973243/30958432-091ef1c4-a435-11e7-9dc4-823b57155cc2.png)